ansible_config.yml

```diff
@@ -0,0 +1,88 @@
+-
```

Comment:
Rename this file to "ansible_playbook", ansible config file is something different: https://docs.ansible.com/ansible/latest/installation_guide/intro_configuration.html#configuration-file